packages/cozy-stack-client/src/NextcloudFilesCollection.js

```diff
@@ -0,0 +1,46 @@
+import DocumentCollection from './DocumentCollection'
+
+const NEXTCLOUD_FILES_DOCTYPE = 'io.cozy.remote.nextcloud.files'
+
+const normalizeDoc = DocumentCollection.normalizeDoctypeJsonApi(
+  NEXTCLOUD_FILES_DOCTYPE
+)
+const normalizeNextcloudFile = (sourceAccount, path) => file => {
+  const extendedAttributes = {
+    ...file.attributes,
+    path: `${path}${path.endsWith('/') ? '' : '/'}${file.attributes.name}`,
```

Comment:
The stack does not return the path like the other files?

Reply:
Nop, but maybe she could do it

Reply:
I think it should, @nono what do you think?

Reply:
Well, it makes the response bigger, and they are quite easy to compute client-side, so I decided to not include them.